build/download-charts.sh

```diff
@@ -61,6 +61,19 @@ function retrieveIstioRelease() {
   )
 }
 
+function sed() {
```

Comment:
I'd rather call this something else, e.g, `sed_wrapper`. This is to avoid confusion with actual sed command.

Reply:
is that a problem? the idea was that you don't have to know this to have it apply

Reply:
I'd prefer not to make things hard to debug. Overwriting system "functions" like this can be dangerous when something break but when we copy and paste the sed command in the terminal it works. Then, after having wasted a lot of time one finds out that there's a sed overwrite function.

Reply:
Done. This is now just a failing function telling you to use `sed_wrap`